packages/experimental/Expander/windows/ReactNativeExpander/ExpanderViewManager.cpp

```diff
@@ -0,0 +1,133 @@
+        nativeProps.Insert(L"chevronBorderPressedBrush", ViewManagerPropertyType::Color);
+
+        return nativeProps.GetView();
+    }
+
+    void ExpanderViewManager::UpdateProperties(xaml::FrameworkElement const& view,
+        IJSValueReader const& propertyMapReader) noexcept {
+        if (auto expanderView = view.try_as<ExpanderView>()) {
+           expanderView->UpdateProperties(propertyMapReader);
+        }
+        else {
+           OutputDebugStringW(L"Type deduction for ExpanderView failed.");
+        }
+    }
+
+    // IViewManagerWithExportedEventTypeConstants
+    ConstantProviderDelegate ExpanderViewManager::ExportedCustomBubblingEventTypeConstants() noexcept {
+        return nullptr;
+    }
+
+    ConstantProviderDelegate ExpanderViewManager::ExportedCustomDirectEventTypeConstants() noexcept {
+        return [](winrt::IJSValueWriter const& constantWriter) {
+            WriteCustomDirectEventTypeConstant(constantWriter, "onChange");
+            WriteCustomDirectEventTypeConstant(constantWriter, L"Collapsing");
+            WriteCustomDirectEventTypeConstant(constantWriter, L"Expanding");
+        };
+    }
+
+    // IViewManagerWithChildren
+    int32_t ExpanderViewManager::ReplaceChild(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& oldChild, winrt::Windows::UI::Xaml::UIElement const& newChild) noexcept {
+        return 0;
+    }
+
+    void ExpanderViewManager::AddView(winrt::Windows::UI::Xaml::FrameworkElement const& parent, winrt::Windows::UI::Xaml::UIElement const& child, int64_t index) noexcept {
+        auto contentPresenter = parent.try_as<winrt::Windows::UI::Xaml::Controls::ContentPresenter>();
+        auto expanderWrapper = contentPresenter.Content();
+        auto expander = expanderWrapper.try_as<Microsoft::UI::Xaml::Controls::Expander>();
+
+        if (auto content = child.try_as<winrt::Windows::UI::Xaml::FrameworkElement>()) {
+            if (index == 0) {
+                expander.Header(content);
+            }
+            else if (index == 1) {
+                expander.Content(content);
+            }
+            else {
+                m_reactContext.CallJSFunction(
+                    L"RCTLog",
+                    L"logToConsole",
+                    MakeJSValueArgWriter("warn", "React Native for Windows does not yet support nesting more than two components under <Expander>.\nThe first component will be the Expander header, and the second will be the Expander content."));
+            }
+        }
+        else {
+            m_reactContext.CallJSFunction(
+                L"RCTLog",
+                L"logToConsole",
+                MakeJSValueArgWriter("warn", "React Native for Windows does not yet support nesting non-FrameworkElement components under <Expander>"));
+        }
+    }
+
+    void ExpanderViewManager::RemoveAllChildren(winrt::Windows::UI::Xaml::FrameworkElement const& parent) noexcept {
+        return;
```

Comment:
I think you can fill these implementations fairly easily, see existing RNW VMs for examples.
Applies to ReplcaeChild and RemoveChildAt too